src/@next/Badge/BadgeStyle.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { S08 } from '../utilities/colors/blue';
+import { B89 } from '../utilities/colors/green';
+import { B95 } from '../utilities/colors/neutral';
+import { S42, S87 } from '../utilities/colors/orange';
+import { B100 } from '../utilities/colors/red';
+import { S75 } from '../utilities/colors/yellow';
+import { space8 } from '../utilities/spacing';
+import { BadgeProps, BadgeStatusType } from './Badge';
+
+const badgeBackgroundColor = {
+  success: B89,
+  informational: S08,
+  warning: S42,
+  critical: B100,
+  promotion: S75,
+  new: S87,
+};
+
+const getBadgeBackgroundColor = (status: BadgeStatusType) => {
+  if (!(status in badgeBackgroundColor)) {
+    return B95;
+  }
+
+  return badgeBackgroundColor[status];
+};
+
+export const BadgeStyle = styled.div`
+  ${({ status }: BadgeProps) => {
+    return { 'background-color': getBadgeBackgroundColor(status) };
+  }}
+  width: fit-content;
+  padding: 2px ${space8};
+  border-radius: 20px;
```

Comment:
there should be https://github.com/glints-dev/glints-aries/blob/c539920df36a3c65a2cc3eb57d243148d6eef0eb/src/%40next/utilities/borderRadius/BorderRadius.ts#L8

Reply:
resolved